ftl/core/preferences.ftl

```diff
@@ -23,7 +23,7 @@ preferences-spacebar-rates-card = Spacebar (or enter) also answers card
 preferences-show-play-buttons-on-cards-with = Show play buttons on cards with audio
 preferences-show-remaining-card-count = Show remaining card count
 preferences-some-settings-will-take-effect-after = Some settings will take effect after you restart Anki.
-preferences-tab-synchronisation = Synchronisation
```

Comment:
Please keep the key (the left part) the same, as otherwise it will make all the different language translators have to retranslate the text. The .ui file shouldn't need to change either.

Reply:
> Please keep the key (the left part) the same
This makes sense why the checks wanted me to update the ui file.

Many thanks for your comment. It should be fine now.

Inadvertently, I made a straightforward PR take longer for you, the opposite of what I had planned.